src/services/flutterwave/client.ts

```
@@ -112,14 +163,16 @@ export class FlutterwaveClient implements FintechProvider {
     recipient: DisburseRecipient,
   ): Promise<DisburseResult> {
     try {
-      const response = await this.client.post("/transfers", {
-        account_bank: recipient.bankCode,
-        account_number: recipient.accountNumber,
-        amount,
-        currency,
-        narration: "ACBU withdrawal",
-        beneficiary_name: recipient.accountName,
-      });
+      const response = await this.requestWrapper(() => 
```

ESLint

[error] Delete `·`

(prettier/prettier)